src/runtime/importhook.cs

```diff
@@ -9,13 +8,46 @@ namespace Python.Runtime
     /// </summary>
     internal static class ImportHook
     {
-        private static IntPtr py_import;
         private static CLRModule root;
-        private static MethodWrapper hook;
         private static IntPtr py_clr_module;
 
         private static IntPtr module_def = IntPtr.Zero;
 
+        private const string LoaderCode = @"
+import importlib.abc
+import sys
+
+class DotNetLoader(importlib.abc.Loader):
+
+    def __init__(self):
+        super(DotNetLoader, self).__init__()
+
+    @classmethod
+    def exec_module(klass, mod):
+        # This method needs to exist.
+        pass
+
+    @classmethod
+    def create_module(klass, spec):
+        import clr
+        return clr._LoadClrModule(spec)
```

Comment:
Maybe we should introduce another module within `clr` for these kind of interface functions. In any case, the exposed functions should (IMO) be "pythonic", in this case something like `clr.internal.load_clr_module`.